tidb-cloud/scale-tidb-cluster.md

```diff
@@ -101,17 +101,24 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
 
 > **Note:**
 >
-> AWS has a cooldown period of node storage changes. If your TiDB cluster is hosted by AWS, after changing the node storage of TiKV or TiFlash, you must wait at least six hours before you can change it again.
+> AWS has a cooldown period of node storage or size changes. If your TiDB cluster is hosted by AWS, after changing the node storage or node size of TiKV or TiFlash, you must wait at least six hours before you can change it again.
```

Comment:
This node should also be mentioned in "Increase node size"

Reply:
added in https://github.com/pingcap/docs/pull/10821/files#r1055154173